### ISSUE TITLE
RTT does not show up on dialer/settings

### ISSUE THREAD

**Issue by StormBorn23** [GrapheneOS/os-issue-tracker#1745]
The Real Time Texting feature is not available in the accessibility menu of the settings, there is a toggle to turn it on in the Google phone app but it doesn't work and the usual options for RTT are not present.

I'm on a pixel 5
Running TP1A.221105.002.2022112500 
Network: Visible
